cloudsmith_cli/core/api/quota.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""API - Quota endpoints."""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import cloudsmith_api
+
+from .. import ratelimits
+from .exceptions import catch_raise_api_exception
+from .init import get_api_client
+
+
+def get_quota_api():
+    """Get the Quota API client."""
+    return get_api_client(cloudsmith_api.QuotaApi)
+
+
+def quota_limits(owner=None, oss=False, **kwargs):
+    """Get Quota for namespace"""
+    client = get_quota_api()
+
+    api_kwargs = {}
+    api_kwargs.update(
+        {param: value for param, value in kwargs.items() if value is not None}
+    )
+
+    if owner and oss:
```

Comment:
Can `owner` be empty/None? Does the validator not prevent this?

Reply:
👍  removed